plugins/modules/helm.py

```diff
@@ -612,17 +635,39 @@ def main():
         if chart_version is not None:
             helm_cmd += " --version=" + chart_version
 
-        if chart_repo_url is not None:
+        if chart_repo_url:
```

Comment:
why this change ? 
In some case it is better to check ``None`` value instead of implicit conversion of str to bool